README.md

```diff
@@ -197,7 +197,7 @@ hasura-backend-plus:
      USER_FIELDS: ''
      S3_ACCESS_KEY_ID: <access-key-here>
      S3_SECRET_ACCESS_KEY: <secret-key-here>
-     S3_ENDPOINT: minio:9000 # Minio port expose below
+     S3_ENDPOINT: http://minio:9000 # Use http only for development
```

Comment:
Not just for development, for production too. The traffic will not go outside the docker network the services are running on. Please remove this comment.